src/Arcus.Observability.Tests.Integration/Serilog/Sinks/ApplicationInsights/EventTests.cs

```diff
@@ -32,7 +32,7 @@ public async Task LogEvent_SinksToApplicationInsights_ResultsInEventTelemetry()
             Dictionary<string, object> telemetryContext = CreateTestTelemetryContext();
 
             // Act
-            Logger.LogEvent(eventName, telemetryContext);
+            Logger.LogCustomEvent(eventName, telemetryContext);
```

Comment:
We should not remove the tests of obsolete functionality. Only new tests should be added that verify the new functionality.

Reply:
Done